config/routes.rb

```diff
@@ -3,4 +3,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  get "users/:user_id/posts", to: "posts#index", as: "user_posts"
+  get "users/:user_id/posts/:id", to: "posts#show", as: "user_post"
+  get "users", to: "users#index", as: "users"
+  get "users/:id", to: "users#show", as: "user"
```

Comment:
- [ ] [OPTIONAL] Kindly write the routes in this way:
```ruby
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

as it's cleaner and easier to read and maintain.

Reply:
Thank you for the helpful suggestion @gemmen29 :smile: